### PR TITLE
Read message webhook

### PR DESCRIPTION
**Pull request by santiagosemhan**
We need to review if the process is ok and I've minor things that I think we need to review. We need the "channel friendly name" in Attributes because otherwise, we have to go back to Twilio again in order to get this name (see current implementation). 

Details: 
- Username is currently a UUID, so I don't know if it's to send the UUID in the body message. 
- LastUpdatedBy, I am assuming that is ModifiedBy Twilio attribute.

### CODE REVIEW

File: src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js [hunk 1]
```diff
@@ -16,10 +18,37 @@
...
+    if (consumers.includes(ModifiedBy)) {
+      const query = new Parse.Query(Parse.User);
+      const [author, reader] = await Promise.all([
+        query.get(From, { useMasterKey: true }),
+        query.get(ModifiedBy, { useMasterKey: true })
+      ]);
+  
+      const channel = await new Twilio().client.chat.services(process.env.TWILIO_SERVICE_SID).channels(ChannelSid).fetch();
+  
+      const body = `${reader.get('username')} read your message in ${channel.friendlyName}`
```
Comment (Benjamindodgson) [inline src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js:35]: User name, would be the "givenName" and "familyName" on the parse user.

Comment (santiagosemhan) [reply src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js:35]: Ok perfect! I'll fix that

File: src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js [hunk 2]
```diff
@@ -16,10 +19,40 @@
...
+      ]);
+
+      const channel = await new Twilio().client.chat
+        .services(process.env.TWILIO_SERVICE_SID)
+        .channels(ChannelSid)
+        .fetch();
+      const username = `${reader.get('givenName')} ${reader.get('familyName')}`;
+      const body = `${username} read your message in ${channel.friendlyName}`;
+      const data = {
+        identifier: ChannelSid + 'read' + reader.id,
```
Comment (Benjamindodgson) [inline src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js:41]: lets add channelId and messageId to the data.

File: src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js [hunk 3]
```diff
@@ -16,10 +19,40 @@
...
-const onMessageUpdated = (request, response) => {
-  return response
-    .status(200)
-    .json({});
+const onMessageUpdated = async (request, response) => {
+  try {
+    let pushStatus = {};
+    let { ChannelSid, Attributes = {}, From, ModifiedBy } = request.body;
+    const { consumers = [] } = Attributes;
+    if (consumers.includes(ModifiedBy)) {
```
Comment (Benjamindodgson) [inline src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js:27]: It should be "lastModifiedBy" according to their docs. 
http://media.twiliocdn.com/sdk/js/chat/releases/3.3.5/docs/Message.html

Comment (santiagosemhan) [reply src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js:27]: Benji! This attribute is not passed to the webhook.
These are the attributes that I can get: https://www.twilio.com/docs/chat/webhook-events#onmessageupdated

**Pull request merged**